composer.json

```diff
@@ -15,6 +15,7 @@
     "ext-PDO": "*",
     "ext-json": "*",
     "ext-sockets": "*",
+    "colinmollenhour/credis": "^1.6.0",
```

Comment:
Not sure on version. I would make it more loose and let Core decide which one to pick. As @kandy suggested, `^1.0.0`  or `~1.0` may work.

Reply:
Since we support Magento `>=2.1.16`, this should be correct constraint: `^1.8.2`

https://github.com/magento/magento2ce/blob/2.1.16/composer.json#L35